pallets/msa/src/weights.rs

```diff
@@ -81,43 +81,43 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Msa MsaKeysOf (r:1 w:1)
 	// Storage: Msa ProviderInfoOf (r:1 w:1)
 	fn create_sponsored_account_with_delegation() -> Weight {
-		(122_000_000 as Weight)
+		(167_000_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	// Storage: Msa KeyInfoOf (r:1 w:0)
 	// Storage: Msa ProviderInfoOf (r:1 w:1)
 	fn remove_delegation_by_provider(s: u32, ) -> Weight {
-		(44_511_000 as Weight)
-			// Standard Error: 2_000
-			.saturating_add((26_000 as Weight).saturating_mul(s as Weight))
+		(69_036_000 as Weight)
```

Comment:
That's weird, I'd think removing data from this struct would lower the weight. I guess it's your hardware.